packages/components/src/CircularProgress/CircularProgress.component.js

```diff
@@ -36,6 +36,7 @@ function CircularProgress({ size, light, percent, className }) {
 		[theme.small]: size === SIZE.small,
 		[theme.default]: size === SIZE.default,
 		[theme.large]: size === SIZE.large,
+		[theme.larger]: size === SIZE.larger,
```

Comment:
Should be named `xlarge` to allow for even bigger sizes.

Reply:
fixed